react-app/src/components/schemaFormUtils.js

```diff
@@ -16,6 +16,7 @@ const uiSchema = {
     classNames: '',
     baseFhirUrl: {
       classNames: 'input-width-limit',
+      'ui:placeholder': 'http://',
```

Comment:
Do you think it makes sense to use `http://example.com` to show it's a full URL? Or is that redundant / not add anything beyond what you have here?

Reply:
I think that's a little much personally. But I'm also not comitted to this change. Happy to revert it if this could benefit from more discussion.

Reply:
No I like it as is!